snakedeploy/scaffold_plugins/common.py

```
@@ -93,11 +105,19 @@ def save_pyproject(pyproject):
         save_pyproject(pyproject)
 
         sp.run(
-            ["pixi", "task", "add", "--feature", "dev", "lint", "ruff check"],
+            ["pixi", "task", "add", "--feature", "dev", "lint", "ruff check src tests"],
```

Ruff (0.15.1)

[error] Starting a process with a partial executable path

(S607)